panoptes_aggregation/panoptes/userify.py

```diff
@@ -0,0 +1,99 @@
+from collections import Iterable
+from json import dumps as jsonify
+from os import environ
+from flask import request
+from panoptes_client import Panoptes, User
+import requests
+from sys import modules
+
+class ConfigurationError(Exception):
+    pass
+
+this = modules[__name__]
```

Comment:
Any reason for defining values on `this` instead of leaving them as `global`s?  They are still accessible to all of the functions without the `this` in front of them.

Reply:
Alternately you could refactor these functions into a class and hold these values on `self` and have `userfy()` be the `__call__` function.

Reply:
no good reason, just not thinking pythonically enough yet i guess :)

Reply:
done